app/models/mixins/miq_provision_quota_mixin.rb

```diff
@@ -330,9 +330,9 @@ def bundle_quota_values(service_resource, result)
   end
 
   def quota_provision_stats(prov_method, options)
-    result = {:count => 0, :memory => 0, :cpu => 0, :storage => 0, :ids => [], :class_name => MiqProvisionRequest.name,
+    result = {:count => 0, :memory => 0, :cpu => 0, :storage => 0, :ids => [], :class_name => "MiqProvisionRequest",
               :active => {
-                :class_name => MiqProvision.name, :ids => [], :storage_by_id => Hash.new { |k, v| k[v] = 0 },
+                :class_name => "MiqProvision", :ids => [], :storage_by_id => Hash.new { |k, v| k[v] = 0 },
```

Comment:
A bonus change that doesn't raise the deprecation but prevents autoload here. and the above two places.